lms/static/styles/components/_ContentSelector.scss

```diff
@@ -0,0 +1,38 @@
+@use "../variables" as var;
+
+.ContentSelector__loading-backdrop {
```

Comment:
These styles have been moved from the `FilePickerApp.scss` file with no changes other than changing the component name from `FilePickerApp` to `ContentSelector`.